ebook_converter/ebooks/odt/input.py

```diff
@@ -84,7 +86,7 @@ def get_css_for_class(self, cls):
                     return rule
 
     def epubify_markup(self, root, log):
-        from ebook_converter.ebooks.oeb.base import XPath, XHTML
+        from ebook_converter.ebooks.oeb.parse_utils import XPath, XHTML
```

Comment:
`XPath` from `parse_utils` have different set of namespaces than the one in `base` module. It might potentially break some other things in namespaces like svg, epub, and others. `XHTML` though is the culprit for the traceback you've potentially bump on.

Instead, just use `parse_utils.XPath(…)` on line  91, and leave line 89 as it was.

Reply:
Ah, and you'll need `from ebook_converter.ebooks.oeb import parse_utils` at the top of the module.